source/routing/raptor_api.cpp

```diff
@@ -806,20 +805,22 @@ get_stop_points( const type::EntryPoint &ep, const type::Data& data,
         }
         const auto admin = data.geo_ref->admins[it_admin->second];
 
+        if (! admin->main_stop_areas.empty()) {
+            for (auto stop_area: admin->main_stop_areas) {
+                for(auto sp : stop_area->stop_point_list) {
+                    const SpIdx sp_idx = SpIdx(*sp);
```

Comment:
minor advice: `const SpIdx sp_idx{*sp};`   `{}` eliminates the most vexing parser and it tells us that `SpIdx` isn't a function.